recipes/r-zerone/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = 'v1.0' %}
```

Comment:
could you just leave the version as '1.0' ? these extra letters can confuse the version ordering of conda.